Makefile

```diff
@@ -195,7 +199,6 @@ install.tools: .install.gitvalidation .install.gometalinter .install.md2man
 	if [ ! -x "$(GOPATH)/bin/go-md2man" ]; then \
 		   go get -d github.com/cpuguy83/go-md2man; \
 		   cd $(GOPATH)/src/github.com/cpuguy83/go-md2man; \
-		   git checkout 20f5889cbdc3c73dbd2862796665e7c465ade7d1; \
 		   go install github.com/cpuguy83/go-md2man; \
 	fi
```

Comment:
Prior this use to be:
go get -u github.com/cpuguy83/go-md2man; \

Are these three lines better than just the one?

Reply:
I'm asking not only for here, but I changed it back to just the single line in Buildah and want to adjust if the 3 are better.

Reply:
I think just `go get -u` without the cd and `go install` is sufficient to get it installed, so I think we should remove the other two lines